e2e/payment/capture.spec.ts

```diff
@@ -0,0 +1,159 @@
+import MercadoPago, { Payment } from '@src/index';
+import fetch from 'node-fetch';
+import { config } from '../e2e.config';
+import type { PaymentCreateData } from '@src/clients/payment/create/types';
+
+describe('IT, capture', () => {
+	test('should capture, return partial transaction_amount passed at the request and match response object', async () => {
+		const client = new MercadoPago({ accessToken: config.access_token, options: { timeout: 5000 } });
```

Comment:
To make the implementation simpler, let's remove the options part.

```suggestion
		const client = new MercadoPago({ accessToken: config.access_token });
```